Add Indexes array to Log Stream Widget

kgrodzicki opened the pull request:
add: Indexes array to LogStreamDefinition(board_widgets.go) and Widget(screen_widgets.go). Update integration test.

According to https://docs.datadoghq.com/dashboards/widgets/log_stream/
logset is Deprecated and indexes should be used. #317

jirikuncar commented on integration/screen_widgets_test.go:
```diff
@@ -497,6 +497,7 @@ func TestWidgets(t *testing.T) {
 			Query:   datadog.String("source:main"),
 			Columns: datadog.String("[\"column_1\",\"column_2\",\"column_3\"]"),
 			Logset:  datadog.String("1234"),
+			Indexes: datadog.String("[\"main\"]"),
```
Can you please create new test case where `Logset` empty?

kgrodzicki replied:
Done 👍

jirikuncar commented:
@kgrodzicki why have you reverted the `make generate` from 4436be3 in f58d2bd?

kgrodzicki commented:
> Thank you for the PR! Can you please resolve conflicts and update the tests? 🙇

@jirikuncar thanks for help with this PR! Have fixed the conflicts, updated tests and rebased commits on top fo the master 🚀

kgrodzicki commented:
> @kgrodzicki why have you reverted the `make generate` from [4436be3](https://github.com/zorkian/go-datadog-api/commit/4436be3c1590890b7806aa5d72e50e7d8d8e1638) in [f58d2bd](https://github.com/zorkian/go-datadog-api/commit/f58d2bdd4829128886ca9de2201a346df9beb050)?

During merge I did change the type for Indexes in screen_widgets.go [5ff5240](https://github.com/zorkian/go-datadog-api/pull/318/commits/5ff524014b02255aba49e3dc7d53366bccb11b0d#diff-cfac79d3891266e939481cbd910092d6) from *string to []*string. In that case the make generate does not generate accessors any more 😅
